data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/master-update.fcc

```diff
@@ -14,6 +14,9 @@ storage:
       path: /etc/kubernetes/static-pod-resources/etcd-member
     - local: etcd-data
       path: /var/lib/etcd
+  directories:
+    - path: /var/lib/etcd
```

Comment:
According to the bug report, for consistency with other installation methods it should only be `/var/lib/etc/member` that is set to `0700`:

> Expected results:
> ```
> $ ls -al /var/lib/etcd
> total 8
> drwxr-xr-x.  3 root root   41 Jun  3 12:02 .
> drwxr-xr-x. 37 root root 4096 Jun  3 12:01 ..
> drwx------.  4 root root   29 Jun  3 12:02 member
> -rw-r--r--.  1 root root  193 Jun  3 12:02 revision.json
> ```